### DISCUSSION TITLE
Dependency Graph and Security Alerts for npm lock file version 3

### DISCUSSION THREAD

**Discussion by avivek** [community/community#49759] [Code Security] [answered]
### Select Topic Area

Bug

### Body

When a repository has security alerts and dependency graphs enabled, no dependency are detected when the package-lock.json is having lock file version 3. 
Node 18 which is LTS comes packed with npm 9 and npm 9 generates lock file version 3 by default.

**Comment by deivid-rodriguez** [comment, marked as answer]
I think this is actually supported since very recently! https://github.blog/changelog/2023-03-10-dependency-graph-and-dependabot-support-npm-v9/
